src/AsyncAPI.Saunter.Generator.Cli/SwashbuckleImport/readme.md

```diff
@@ -0,0 +1,3 @@
+This code is taken from [Swashbuckle.AspNetCore.Cli](https://github.com/domaindrivendev/Swashbuckle.AspNetCore/tree/master/src/Swashbuckle.AspNetCore.Cli)
+
+Since Swashbuckle.AspNetCore.Cli is delivered as a tool, code cannot be reference through Nuget. 
```

Comment:
this is a very bad practice, we will not be able to support it.

there are other ways to reuse their code, calling their own utility, for example, and linking to it